packages/tooltip/src/Tooltip.js

```diff
@@ -5,15 +5,43 @@ import values from "object.values";
 import { Container, StyledTooltip } from "./styledComponents/Tooltip";
 import alignments from "./constants/alignments";
 
-const Tooltip = props => {
-  const content = props.children || props.text;
-  return (
-    <Container>
-      {content && <StyledTooltip {...props}>{content}</StyledTooltip>}
-    </Container>
-  );
+export const getAutoAlign = bodyWidth => containerRef => align => {
+  if (align !== alignments.AUTO) return align;
+  if (containerRef) {
+    const { x } = containerRef.getBoundingClientRect();
+    if (x > bodyWidth / 2) return alignments.RIGHT;
```

Comment:
Some comments would be nice to help us understand what's going on... (why `bodyWith / 2` and so on)